plugins/catalog-react/src/apis/StarredEntitiesApi/migration.ts

```diff
@@ -41,8 +41,7 @@ export async function performMigrationToTheNewBucket({
     // nothing to do
     return;
   }
-
-  const targetEntities = new Set(target.get('entityRefs') ?? []);
+  const targetEntities = new Set(target.get<string[]>('entityRefs') ?? []);
```

Comment:
ah and this needs no changeset because it doesn't affect any interfaces nor the published artifact i guess

Reply:
yep!